### PR TITLE
Fixes #23642 - Update 1.16 - 1.19 installer manuals

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by ekohl** [comment]
Looks like I forgot the digitalocean exception and that link is now a 404.

**Review by dgoetz — COMMENTED** [review]
Except from the Digital Ocean link you already mentioned, it looks good. Some values lose their defaults or another small bit of information, but I think nothing important. 

The most prominent example is `Add a file to /etc/sudoers.d (true) or uses augeas (false)` which will be shortened to `Add a file to /etc/sudoers.d (true)` where the or part was also not perfect as it would only tell an experienced user that it will use augeas to directly add a sudo rule to /etc/sudoers.

Thanks!

**Comment by ekohl** [comment]
That's because it was changed in https://github.com/theforeman/puppet-foreman_proxy/blob/fde8d9b9547b1dc50b6010616b4aaf5bde51fbf4/manifests/init.pp#L68 so that's the actual help text. The fact it was changed in https://github.com/theforeman/puppet-foreman_proxy/commit/72dca5a10f409e80c32c1d416ae9bf0d61e44972 (almost 2 years old) suggests we haven't updated these help texts in a LONG time.

As you can see I've scripted most of this part already and will be updating the release procedure so we're reminded every time.

**Comment by ekohl** [comment]
Updated. I've improved the logic in the script to autodetect plugin manuals and added an exception for digitalocean.

File: scripts/installer/installer-enable-to-table [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+PREFIXES = {
+    '--[no-]enable-foreman-plugin-': 'foreman_',
+    '--[no-]enable-foreman-proxy-plugin-': 'smart_proxy_',
+}
+
+
+PLUGINS = {
+    'digitalocean': {
+        'url': 'https://github.com/theforeman/foreman-digitalocean',
```
Comment (ekohl) [inline scripts/installer/installer-enable-to-table:16]: https://community.theforeman.org/t/move-foreman-digitalocean-plugin/11845

**Review by tbrisker — APPROVED** [review]
Thanks @ekohl and @dgoetz !

**Pull request merged**